arbnode/transaction_streamer.go

```diff
@@ -1428,22 +1409,6 @@ func (s *TransactionStreamer) getEspressoSubmittedTxns() ([]arbutil.SubmittedEsp
 	return tx, nil
 }
 
-func (s *TransactionStreamer) getLastConfirmedPos() (*arbutil.MessageIndex, error) {
```

Comment:
`LastConfirmedPos` was used to indicate the position of finalized messages. Now we are using the espresso streamer, which should have validated when messages being enqueued. So we don't need this anymore.